orchestrators/ProcurementOperations/src/Services/PaymentProcessingService.php

```
@@ -113,622 +77,175 @@ public function createBatch(
-            str_pad((string) $totalDebit, 12, '0', STR_PAD_LEFT),
-            str_pad((string) $totalCredit, 12, '0', STR_PAD_LEFT)
-        );
-
-        // Pad to block size
-        while (count($lines) % 10 !== 0) {
-            $lines[] = str_repeat('9', 94);
-        }
-
-        return implode("\n", $lines);
+    public function estimateBankFees(PaymentBatchData $batch): Money {
+        return Money::of(0, 'USD');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\Common\ValueObjects\Money' in method 'estimateBankFees'. (undefined)

(StaticAccess)